docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -0,0 +1,151 @@
+# Tizen Window System Shell
+
+
+The Tizen window system shell (TZSH) is an interface for manipulating windows of the sytem graphical user interfaces (GUI) services such as the quickpanel and the softkey. It provides C libraries that allow developers to control windows of system GUI services and be notified state changes of them within an application.
```

Comment:
The Tizen window system shell (TZSH) is an interface for manipulating windows of the system graphical user interfaces (GUI) services such as the quick-panel and the soft-key. It provides C libraries that allows you to control windows of system GUI services and get notifications about the state changes of each window within an application.

Reply:
@nawab-samsung
I agree with @mijin85cho's opinion. Same to above comment, quickpanel and softkey are more correct words in the Tizen.

And I think "C libraries that allow" is correct.

@mijin85cho
Thanks I've changed this as your comment. TZSH and GUI.

Reply:
@gl-lee  I too am in agreement with @mijin85cho s comment, but why i changed there was a reason behind that too i.e subject verb agreement.

Reply:
Isn't "C libraries" plural? I thought "C libraries" as a subject.